src/App.tsx

```diff
@@ -10,31 +11,27 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, selectedTabIdEdit] = useState('tab-1');
+  const selectedForm = tabs.filter((tab) => tab.id === selectedTabId);
```

Comment:
seems that currentTab would fit better. But it's up to you. And better to use the find method here. You need only one tab
```suggestion
  const currentTab = tabs.find((tab) => tab.id === selectedTabId) || tabs[0];
```